### PR TITLE
Adds an option to pass the value 

### PR DESCRIPTION
**Pull request by ThinkTankShark**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -174,6 +162,7 @@ TagInput.propTypes = {
   optionGroupTitles:  PropTypes.array,
 
   httpHeaders:        PropTypes.object,
+  value:              PropTypes.instanceOf(Immutable.List),
```
Comment (isuttell) [inline src/index.js]: This should be required shouldn't it?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ship-components-tag-input",
-  "version": "0.3.0",
+  "version": "1.0.0",
```
Comment (isuttell) [inline package.json]: I'm not sure we're stable enough for 1.0.0 yet

File: src/Dropdown.css [hunk 3]
```diff
@@ -31,17 +31,23 @@
...
 
-.option,
-.option-btn {
-    background-color: #fff;
-    color: #333;
+    padding: 8px 24px;
+    white-space: nowrap;
+    cursor: pointer;
+    overflow: hidden;
+    -webkit-transition: 0.3s ease all;
```
Comment (isuttell) [inline src/Dropdown.css]: We don't need the prefix, we use autoprefixr for that

File: src/index.js [hunk 4]
```diff
@@ -27,17 +26,13 @@ export default class TagInput extends React.Component {
    * @memberof TagInput
    */
   handleSelectItem(item) {
-    let selection = this.state.selection.slice(0);
-    selection = selection.push(item);
+    let newValue = this.props.value.slice(0);
```
Comment (isuttell) [inline src/index.js]: We don't need to clone here since we're using immutables

File: src/index.js [hunk 5]
```diff
@@ -48,25 +43,18 @@ export default class TagInput extends React.Component {
...
-        // Sending the tags to parent
-        if (typeof this.props.onChange === 'function') {
-          this.props.onChange(this.state.selection);
-        }
-      });
+      newValue = newValue.splice(index, 1);
     }
 
     // Sending the tags to parent
     if (typeof this.props.onChange === 'function') {
```
Comment (isuttell) [inline src/index.js]: This should probably be required now

**Comment by ThinkTankShark** [comment]
Moved to this PR #8

**Pull request closed without merging**